atd-vze/src/views/Crashes/CrashDiagram.js

```diff
@@ -119,27 +124,27 @@ const CrashDiagram = props => {
               </>
             )}
           </TransformWrapper>
-        ) : props.isTempRecord ? (
-          <div className="mt-2">
-            CR-3 PDFs, diagrams and narratives are not available for temporary
-            records. Using the case id, check the{" "}
-            <a
-              href={"https://cris.dot.state.tx.us/"}
-              target={"_blank"}
-              rel="noopener noreferrer"
-            >
-              CRIS website
-            </a>{" "}
-            for the latest status of this crash.
-          </div>
         ) : (
+          // ) : props.isTempRecord ? (
```

Comment:
just commenting this out for now until we figure out how we are going to handle temp records in the new db